### ISSUE TITLE
Register domain for external access

### ISSUE THREAD

**Issue by hwakabh** [hwakabh/bennu-official#119]
# AsIs
Currently we have to specify IP address to access application running on GKE.
This address would be not fixed one, as this will be changed if k8s node would be restarted for some reason like maintenance or upgrade, so that we have to use domain name.
Since we have already had `bennu-official.page` domain for this application, we can use this domain for external access to Django application in production.

# ToDo
- Determine implementations/solutions
  - GKE Ingress: Easiest way to expose with custom domain, but expensive
  - NGINX Ingress controller
  - Contour (Ingress controller based on Envoy)
  - Static Reserved IP with `type: LoadBalancer`

# (Optional) Acceptance criteria
We can access application with custom domain, `bennu-official.page`

# (Optional) Additional contexts
N/A

**Comment by hwakabh** [comment]
As implemented deployment/nginx onto GKE in #82 , and installed kind: Ingress with using ingress-nginx-controller in #114 , what we have to do in this issue is to only apply custom domain, `bennu-official.page` to public IP address on GKE.

Since `type: LoadBalancer` service have been created as external address when we use ingress-nginx-controller, we have to associate `EXTERNAL-IP` of this service resource with our custom domain.
Note that when we would create Ingress resource at k8s cluster, where ingress-nginx-controller is installed, `ADDRESS` of ingress will be exactly same as `EXTERNAL-IP` of `namespace/ingress-nginx/services/ingress-nginx-controller`.
Also note that when we would create `type: LoadBalancer` service resource on GKE, the **ephemeral** IP address will be automatically generated and associated to service, so we have to make this as fixed one.

For creating fixed IP address, we should run the following command.
```shell
# Create `regional` fixed IP, where as there is also `global` fixed IP in Google Cloud
gcloud compute addresses create ${name_of_address_instance} --region=us-central1
```

Then, added this provisioned IP address as A record in Google Domains admin menu.

**Comment by hwakabh** [comment]
Evidence:
```shell
24-02-01 22:53:23 ~  % gcloud compute addresses list
NAME          ADDRESS/RANGE   TYPE      PURPOSE  NETWORK  REGION       SUBNET  STATUS
minio-public  34.121.252.242  EXTERNAL                    us-central1          IN_USE

24-02-01 22:55:39 ~  % nslookup bennu-official.page -type=A 8.8.8.8
Server:		8.8.8.8
Address:	8.8.8.8#53

Non-authoritative answer:
Name:	bennu-official.page
Address: 34.121.252.242
```
